src/compiler/diagnosticMessages.json

```diff
@@ -2347,6 +2347,14 @@
         "category": "Error",
         "code": 7026
     },
+    "JSX element implicitly has type 'any' because no interface 'JSX.{0}' exists": {
+        "category": "Error",
+        "code": 7026
+    },
+    "Array element at index {0} implicitly has an '{1}' type.": {

```

Comment:
You're right, I think I'll just remove it and remove the associated logic. I think it will be too hard to hit this error.